Exception in thread "main" MemoryError: NullPointerException in method call
    at com.example.src.test.mock_35.py.process(mock_35.py:115)
    at com.example.processors.DataProcessor.processUserData(DataProcessor.java:78)
    at com.example.Main.main(Main.java:45)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)